msu/hooks/ai/tactical/agent.nut

```diff
@@ -0,0 +1,47 @@
+::mods_hookNewObject("ai/tactical/agent", function(o) {
```

Comment:
Could you do that and explain why cuz I'm not sure what this hook was for.

Reply:
This was to prevent the ai from thinking while using a move skill e.g. rotation to hopefully prevent the ai-based rotation crash that happens in think(). However, this doesn't fix it.